python/external_config/scripts/external_runner.py

```diff
@@ -304,6 +308,16 @@ def main():
     """
     task_runner = QtTaskRunner(main)
 
+    # For qt5, we may get this error:
+    #
+    # RuntimeError: Qt WebEngine seems to be initialized from a plugin.
+    # Please set Qt::AA_ShareOpenGLContexts using QCoreApplication::setAttribute
+    # before constructing QGuiApplication.
+    if hasattr(qt_importer.QtCore.Qt, "AA_ShareOpenGLContexts"):
```

Comment:
Do we need a `QtGui` application here? Could this be solved by using a `QtCore` application?

Reply:
We always launch a `QTGui` application right now - perhaps in the future we can restrict this, but for the time being that seems to be the simplest approach for launching